tox.ini

```diff
@@ -33,6 +33,25 @@ commands =
     {toxinidir}/ci/build_ndcore.sh {toxinidir}/neurodamus/data {toxworkdir}
     sh -c ". {toxworkdir}/.envfile && pytest -x --cov=neurodamus --cov-report=term-missing --forked {posargs:tests/unit}"
 
+[testenv:unit-mpi]
+# Please you need to spack load neurodamus-models model=neocortex +coreneuron or use an equivalent spack virtual env
+; passenv = *
```

Comment:
needed?

Reply:
no, again debug code. I was sloppy in removing stuff around sorry

Reply:
np, just wanted to make sure, happens to everyone